README.md

```diff
@@ -13,39 +14,81 @@ At this time there are no web server-side components.
 
 ## DaoStack-Arc-Js
 
-Alchemy uses a library of reusable and configurable Ethereum smart contracts called "DaoStack-Arc-Js".  You can see all of the source code for those contracts and even contribute to the project [here](https://github.com/daostack/arc-js).
+Alchemy uses a library of reusable and configurable Ethereum smart contracts called "DaoStack-Arc-Js". You can see all of the source code for those contracts and even contribute to the project [here](https://github.com/daostack/arc-js).
 
 ## See It Live
-You can browse to the latest release at [http://daostack.azurewebsites.net](http://daostack.azurewebsites.net).  This deployment of Alchemy assumes you are either running a Kovan chain locally (listening on localhost:8485), or you are using Chrome and have an extension such as MetaMask that enables you to connect to a Kovan node.
 
-## Building The Code
+You can browse to the latest release at [http://daostack.azurewebsites.net](http://daostack.azurewebsites.net). This deployment of Alchemy assumes you are either running a Kovan chain locally (listening on localhost:8485), or you are using Chrome and have an extension such as MetaMask that enables you to connect to a Kovan node.
+
+## Build and Run
 
-To build the code, follow these steps.
+The file package-scripts.js contains a number of scripts to help you build, run and debug the application.  To build the code, follow these steps:
 
-1. Ensure that [NodeJS](http://nodejs.org/) is installed.
+1. Ensure that [NodeJS](http://nodejs.org/), v8.0.0 or greater, is installed.
 2. From the project folder, install the packages:
 
-  ```shell
-    npm install
-  ```
-3. build the application:
-  ```shell
-    npm start webpack.build.production
-  ```
+```shell
+  npm install
+```
 
-4. start the application:
-  ```shell
-    npm start serve
-  ```
+3. Fire up ganache (testrpc) in a separate window:
+
+```shell
+npm start arc-js.ganache.runAsync
```

Comment:
I would suggest to use fire up a none cli ganache in this case : https://github.com/trufflesuite/ganache
and (npm install and npm start).
For running on ganache there is no meaning to link the addresses  to https://ganache.etherscan.io/ as the code currently doing.
For the case of ganache the link should be something like : HTTP://127.0.0.1:7545/<address> so one can get a real view of the address.

Reply:
@orenyodfat Sorry, I don't understand what you're saying here:

> I would suggest to use fire up a none cli ganache in this case : 
> https://github.com/trufflesuite/ganache
>and (npm install and npm start).

Are you suggesting that `npm start` with no arguments fire up ganache (`arc-js.ganache.runAsync`) and migrate the contracts into it (`arc-js.migratContracts`)?  (I think that is doable)

Reply:
@orenyodfat Great suggestion:

> For running on ganache there is no meaning to link the addresses to https://ganache.etherscan.io/ as the code currently doing.
> For the case of ganache the link should be something like : HTTP://127.0.0.1:7545/

Reply:
@orenyodfat 

> It is very handful and useful to add an automated tests which try to cover code you submit.
> For the one who submit the pr and the one who reviewing it.
>It would also catch simple issues like failing at "npm install"

At minimum, I can create a travis script that will test the scripts I've added in this PR by running ganache and doing a migration, the same thing that the arc-js travis test script does.  That will be a start on fleshing out more coding tests later one.

Reply:
@orenyodfat 

> For running on ganache there is no meaning to link the addresses to https://ganache.etherscan.io/ as the code currently doing.
> For the case of ganache the link should be something like : HTTP://127.0.0.1:7545/

I am not able to find any way to explore transactions on ganache that is out-of-the-box.

Reply:
@dkent600 
"I am not able to find any way to explore transactions on ganache that is out-of-the-box."
try to use ganache none cli .

Reply:
@orenyodfat 

OK, you mean _non_ cli ganache as in a GUI explorer for ganache.  I didn't know they had such an app.

You are suggesting that the ganache explorer be:

1) something that can display information about a transaction via the url at port 7545
2) launched by the script `arc-js.ganache.runAsync`

It is not trivial, at the least, to install and build this application for the user.  I think the suggestion is out-of-scope for this PR.  The cli ganache works.

So for the transaction links that are currently broken with ganache, I can change them to create the non-cli app url.  The url will not work unless the user happens to have the explorer running.  Or I can just make the urls not hot if the demo is targetting ganache.

Which option do you prefer?

Reply:
> So for the transaction links that are currently broken with ganache, I can change them to  create the non-cli app url. The url will not work unless the user happens to have the explorer running. Or I can just make the urls not hot if the demo is targetting ganache.
Which option do you prefer? 

I prefere to make this links meaningful (hot).
Not sure why you cannot fire up the none-cli ganache app...(browser based) and migrate to that?

Reply:
@orenyodfat 

> Not sure why you cannot fire up the none-cli ganache app...(browser based) and migrate to that?

It is because the application has to be installed, just like with parity, and that is not a simple matter of running npm.

Changing the url is an easy and low-risk change, I'll go ahead and roll it into this PR.

Reply:
>It is because the application has to be installed, just like with parity, and that is not a simple matter of running npm.

it can be fired up by npm install & npm start . (this is how I am bringing that up)

Reply:
You want `npm install` in what context?

Reply:
@orenyodfat 

> I prefer to make this links meaningful (hot).

If the ganache GUI client supports automation via URL requests like this, I have not found a way to get it to work, neither by experimentation nor documentation.  Any suggestions?